docs/DeveloperGuide.md

```diff
@@ -11,7 +11,7 @@
 * [Appendix B: Use Cases](#appendix-b--use-cases)
 * [Appendix C: Non Functional Requirements](#appendix-c--non-functional-requirements)
 * [Appendix D: Glossary](#appendix-d--glossary)
-* [Appendix E : Product Survey](#appendix-e-product-survey)
+* [Appendix E: Product Survey](#appendix-e--product-survey)
```

Comment:
@team ! Great job and Great progress! Will touch on Dev guide in time to come!!! Meanwhile do come to me if any clarification is needed ~ or simply tag me here and email me (since i've tons of github PRs i might missed the email from github :) )